omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonAttributeResource1_8.java

```diff
@@ -96,9 +97,13 @@ public void setHydratedObject(PersonAttribute personAttribute, String attributab
 			throw new APIException("Could not convert value to Attributable", e);
 		}
 	}
-	
+
 	@PropertySetter("value")
 	public void setValue(PersonAttribute personAttribute, String value) {
+		if (setLocationIfUUID(personAttribute, value)) {
+			return;
+		}
```

Comment:
I think I'd probably just inline `setLocationIfUUID()` here. I don't see a lot of value in having a separate method.